docs/user/terms.rst

```diff
@@ -5,41 +5,55 @@ Glossary
 
     digraph {
       rankdir=RL;
-      { Service Prometheus} -> Shard;
       { Rule Notifier Project} -> Service;
+      { Project Prometheus} -> Shard;
       { Rule Notifier Exporter URL Farm} -> Project;
       Host -> Farm;
-      { rank=same Project Rule };
+      { rank=same Notifier Rule };
    }
 
 
 Shard
 -----
 
-Shards are the top level object in Promgen which contain both Services and
-Prometheus servers
+Shards are a collection of Prometheus servers monitoring the same targets.
 
 Service
 -------
 
-A service in Prometheus is a group of related Projects and are assigned to a
-specific shard
+A service in Prometheus is a group of related Projects and Rules and are the
+main top-level object.
```

Comment:
What would you recommend then?

Reply:
> What would you recommend then?

I am not sure. Maybe this?

"A Service in Prometheus is a group of related Projects and Rules. Those three are the
main top-level objects."

Are there other top-level objects that are not considered "main"?

Reply:
I realized I need to fix this anyways, because `service` is a Promgen concept and not Prometheus.

> A service in Promgen is a group of related Projects and Rules.
> Service is typically the top-level object in Promgen.